governance/polls/Vault Compensation Poll 3

```diff
@@ -0,0 +1,54 @@
+
+title: Vault Compensation Poll
+summary: Signal your support for vault compensation
```

Comment:
Signal your support or opposition for compensation of Black Thursday victims?

Reply:
Updated the wording, let me know what you think.